lib/apia/open_api/objects/parameters.rb

```diff
@@ -69,13 +69,29 @@ def add_to_spec
               schema: generate_scalar_schema(@argument)
             }
             param[:description] = @argument.description if @argument.description.present?
+
+            add_pagination_params(param)
+
             param[:required] = true if @argument.required?
             add_to_parameters(param)
           end
         end
 
         private
 
+        def add_pagination_params(param)
+          if param[:name] == "page"
+            param[:description] = "The page number to request. If not provided, the first page will be returned."
+            param[:schema][:default] = 1
+            param[:schema][:minimum] = 1
+          elsif param[:name] == "per_page"
+            param[:description] =
+              "The number of items to return per page. If not provided, the default value will be used."
+            param[:schema][:default] = 30
```

Comment:
Can we get this `30` value from Apia itself rather than hard-code it?

Reply:
Also I don't see this in the openapi.json spec fixture... can we add this to the example so we get it covered by the tests?